### ISSUE TITLE
Self-signed certificate rejected

### ISSUE THREAD

**Issue by sempervictus** [inexio/eve-ng-cli-manager#1]
Eve-ng's virtual appliance comes with a self-signed cert. After a bit of digging, it looks like the resty TLS context needs to be modified, and since every API call is a separate function in the CLI leaf commands, its a bit awkward to do from this source tree. Is there any way to pass an env var/yaml config option to permit self-signed certs?

**Comment by janisgross** [comment]
Yes, I think it should be possible to have an env var/yaml parameter that could change the resty behaviour, to allow self signed certs.

I will have a look into implementing such a feature.

In the meantime, could you maybe elaborate a little bit more on your specific problem, give an example of your certificate ?

**Comment by sempervictus** [comment]
The specific problem i'm seeing is that invocation of any command returns `ERR Error during login` when passing `--config ~/.eve_ng.yaml` which specifies `url:` as the HTTPS endpoint of a private eve-in-a-box KVM instance. The certificate on that endpoint is a self-signed one generated by the EVE appliance, so it doesn't get verified by resty. Credentials in the YAML are the same ones used to log into the web UI.
`
